src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -50,8 +50,13 @@
     #set ($discard = $response.setContentLength($svg.length()))
     #set ($discard = $response.writer.write($svg))
   #elseif ($xcontext.action == 'export')
+    #set ($fileName = $doc.name)
+    #if ($fileName == 'WebHome')
+      #set ($fileName = $doc.documentReference.lastSpaceReference.name)
+    #end
+    #set ($fileName = "${fileName}.png")
     {{html clean="false"}}
-    &lt;img src="$doc.getURL('get', "data=svg&amp;v=$!doc.version")" alt="$escapetool.xml($doc.plainTitle)" /&gt;
+    &lt;img src="$doc.getAttachmentURL($fileName, 'download', "v=$!doc.version")" alt="$escapetool.xml($doc.plainTitle)" /&gt;
```

Comment:
I think it's worth checking if the attachment is present and if not then fallback on the SVG export (the previous behavior).